contracts/libraries/LibDiamond.sol

```diff
@@ -102,7 +103,8 @@ library LibDiamond {
             address oldFacetAddress = ds.selectorToFacetAndPosition[selector].facetAddress;
             require(oldFacetAddress == address(0), "LibDiamondCut: Can't add function that already exists");
             addFunction(ds, selector, selectorPosition, _facetAddress);
-            selectorPosition++;
+            ++selectorPosition;
+            unchecked { ++selectorIndex; }
```

Comment:
> for (uint256 selectorIndex; selectorIndex < _functionSelectors.length; selectorIndex++)

`selectorPosition` is now being incremented twice in each iteration of the loop, instead of once.